components/Modules/VsBrDropdownWithSearch.vue

```
@@ -0,0 +1,59 @@
+
+<script setup lang="ts">
+import { computed, inject } from 'vue';
+
+import {
+    VsCheckbox,
+    VsDropdown,
+    VsDropdownItem,
+} from '@visitscotland/component-library/components';
+import type { SearchFilterCategory } from '~/types/types';
+
+import useSearchStore from '~/stores/searchStore';
```

ESLint

[error] Missing file extension for "~/stores/searchStore"

(import/extensions)